packages/fxa-graphql-api/src/scripts/must-change-password.ts

```diff
@@ -36,21 +36,64 @@ const logger = mozlog(config.log)('must-change-password');
 const requiredOptions = ['input'];
 
 program
-  .option('-i, --input <filename>', 'JSON input file')
+  .option('-e, --emails [emails]', 'Email addresses')
+  .option('-u, --uids [uids]', 'User IDs')
+  .option('-i, --input <filename>', 'Input filename from which to read input')
   .option(
     '-b, --batch-size <count>',
     'How many records to update in a single db UPDATE.'
   )
   .parse(process.argv);
 
-function checkRequiredOption(optionName: string) {
-  if (!program[optionName]) {
-    console.error(`--${optionName} required`);
+if(!program.input) {
+  console.error(`-i, --input required`);
+  process.exit(1);
+} else if(!program.emails && !program.uids) {
+  console.error('One of `emails` or `uids` must be specified');
+  process.exit(1);
+} else if(program.emails && program.uids) {
+  console.error('Only one of `emails` or `uids` can be specified, not both');
+  process.exit(1);
+}
+
+let emails = [];
+let uids = [];
+
+if (program.emails) {
+  emails = getItems('emails');
+} else if (program.uids) {
+  uids = getItems('uids');
+}
+
+if (!emails.length && !uids.length) {
+  console.error('No `emails` or `uids` found inside the specified file');
+  process.exit(1);
+}
+
+function getItems(type) {
```

Comment:
It doesn't look like `type` is used, should it be?

Reply:
@dannycoates I agree that it is safe to remove type from both the files `must-enhance-password.ts` and `must-reset.js`

Reply:
Shall I also rebase my commits.

Reply:
> One more thing that there is no such test file for `must-enhance-password.ts` so shall I make a new issue for this.

sure, you're welcome to :)

Reply:
@dannycoates So is the pr ready to be merged?